app/src/features/HjelpeTekst.tsx

```diff
@@ -0,0 +1,52 @@
+import type { AlertProps, ReadMoreProps } from "@navikt/ds-react";
+import { Alert } from "@navikt/ds-react";
+import { ReadMore, Switch } from "@navikt/ds-react";
+import { useNavigate, useSearch } from "@tanstack/react-router";
+
+import { setLocalStorageItem } from "~/features/local-storage-utils.ts";
+
+export const VIS_HJELPETEKSTER_KEY = "vis-hjelpe-tekster";
+
+export function HjelpeTekstMasterSwitch() {
```

Comment:
Digger navnet 😄 Men kanskje HjelpetekstToggle er lettere å forstå for andre etter oss?